sorc/ncep_post.fd/SURFCE.f

```diff
@@ -1443,14 +1445,8 @@ SUBROUTINE SURFCE
                Q1D(I,J) = QSHLTR(I,J)
              ENDDO
            ENDDO
-!            CALL CALRH(PSHLTR,TSHLTR,QSHLTR,EGRID1)
-           IF(MODELNAME == 'GFS')THEN
-             CALL CALRH_GFS(P1D,T1D,Q1D,EGRID1(1,jsta))
-           ELSEIF(MODELNAME == 'RAPR')THEN
-             CALL CALRH_GSD(P1D,T1D,Q1D,EGRID1(1,jsta))
-           ELSE
-             CALL CALRH(P1D,T1D,Q1D,EGRID1(1,jsta))
-           END IF
+
+           CALL CALRH(P1D,T1D,Q1D,EGRID1(:,jsta:jend))
```

Comment:
@JesseMeng-NOAA Could you keep the original style like
CALL CALRH(P1D,T1D,Q1D,EGRID1(1,jsta))